schemas/veres-one-validator.js

```diff
@@ -125,26 +126,33 @@ const publicKey = {
     },
     type: {
       type: 'string',
-      enum: ['RsaVerificationKey2018', 'Ed25519VerificationKey2018'],
+      enum: ['Ed25519VerificationKey2020', 'X25519KeyAgreementKey2020'],
     },
     controller: did(),
-    // FIXME: make schema require this for RsaVerificationKey2018
-    publicKeyPem: schemas.publicKeyPem(),
-    // FIXME: make schema require this for Ed25519VerificationKey2018
-    publicKeyBase58: {
-      type: 'string',
+    // FIXME: make schema require this for Ed25519VerificationKey2020
+    publicKeyMultibase: {
+      type: 'string'
     }
   },
 };
 
 const didDocumentContext = {
   type: 'array',
-  items: [{
-    const: constants.DID_CONTEXT_URL
-  }, {
-    const: constants.VERES_ONE_CONTEXT_V1_URL
-  }],
-  maxItems: 2,
+  items: {
+    // FIXME: Force first two items to be DID and V1 context?
```

Comment:
Yes, this should be forced.

Reply:
I can handle this in my current validators branch if we want to put this item here: https://github.com/veres-one/veres-one-validator/pull/38

Reply:
Yes, please @aljones15

Reply:
this was addressed here: https://github.com/veres-one/veres-one-validator/pull/35/commits/2b8c0b5b6ba9d35f8ba9ee4fb5054d7672b9c098

Reply:
Hmm, still an issue, please see my comment here: https://github.com/veres-one/veres-one-validator/commit/2b8c0b5b6ba9d35f8ba9ee4fb5054d7672b9c098#r53383105